src/Baka/Support/Image.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Baka\Support;
+
+use Illuminate\Support\Facades\Storage;
+
+class Image
+{
+    public static function downloadFileToLocalDisk(string $url, string $path): string
+    {
+        $file = file_get_contents($url);
```

Comment:
@FredPeal check to see if laravel or spatie have better package than just get docs